src/generic/components/ecl-navigation/ecl-navigation-side/ecl-navigation-side.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Side navigation related behaviors.
+ */
+
+import stickybits from 'stickybits';
+
+/**
+ * @param {object} options Object containing configuration overrides
+ */
+export const navigationSide = ({
+  stickySelector: stickySelector = '.ecl-navigation-side__toggle',
+} = {}) => {
+  // SUPPORTS
+  if (
+    !('querySelector' in document) ||
```

Comment:
I'm not sure whether all the checks are needed in this component, as actually `getElementsByClassName` is used as a query selector. The `querySelector` and `classList` are not really used. Not a big problem.